src/util/SearchUtils.ts

```diff
@@ -0,0 +1,69 @@
+import _ from 'lodash';
+
+export function uniformSearchText(keyword: string) {
```

Comment:
We should probably add some comment for this function. Also `normalizeSearchText` might be a better name.